### ISSUE TITLE
Dockerform generates wrong ports for nodes

### ISSUE THREAD

**Issue by IlyaSavich** [corda/corda-gradle-plugins#301]
Hi, trying to generate nodes using Dockerform and getting weird result
Here my config
```groovy
task deployNodes(type: net.corda.plugins.Dockerform, dependsOn: ['jar']) {
    nodeDefaults {
        projectCordapp {
            deploy = false
        }
        cordapp project(':contracts')
        cordapp project(':workflows')
    }
    node {
        name "O=Notary,L=London,C=GB"
        notary = [validating : false]
        p2pPort 10002
        rpcSettings {
            address("localhost:10003")
            adminAddress("localhost:10043")
        }
    }
    node {
        name "O=Users,L=London,C=GB"
        p2pPort 10005
        rpcSettings {
            address("localhost:10006")
            adminAddress("localhost:10046")
        }
        rpcUsers = [[ user: "user1", "password": "test", "permissions": ["ALL"]]]
    }
}
```
Running ./gradlew deplyNodes
This I've got for Notary node
```
// Notary/node.conf

p2pAddress="notary:10002"
rpcSettings {
    address="notary:10003"
    adminAddress="notary:10005"
}
```
And this for Users node
```
// Users/node.conf

p2pAddress="users:10005"
rpcSettings {
    address="users:10003"
    adminAddress="users:10005"
}
```
So it leads to error when trying to run nodes, so I need manually fix the issue. Interesting fact that I don't get such an error when using `Cordform`

**Comment by chrisr3** [comment]
Hi, thanks for the report. Could you also let us know which versions of the `cordformation` plugin and Gradle you are using please? Thanks.

**Comment by IlyaSavich** [comment]
cordformation: 5.0.4
gradle: 5.4.1

**Comment by chrisr3** [comment]
Fixed by #304 .
